proto/ignition/msgs/navsat_sensor.proto

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "Protos";
```

Comment:
Should this be `NavSatSensorProtos`?

Reply:
Good eye! This was actually copy-pasted, there are other messages with the same problem:

```
$ grep -r "\"Protos\"" proto/ignition/msgs/
proto/ignition/msgs/sensor_noise.proto:option java_outer_classname = "Protos";
proto/ignition/msgs/polylinegeom.proto:option java_outer_classname = "Protos";
proto/ignition/msgs/gps_sensor.proto:option java_outer_classname = "Protos";
```

I'm not sure what this affects and whether fixing this could break ABI :thinking: